subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,15 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files) found in inputs/outputs of tasks are handled gracefully from now on: similarly to missing files.
+
+If the output contains an unavailable file - not produced by the task -, it will be ignored.
```

Comment:
I think everything from here to the end can be removed.

Reply:
Okay, if you think so